### ISSUE TITLE
sumo not working for Highway Driving examples on Mac OS 10.3.6 with Macports

### ISSUE THREAD

**Issue by WolfgangFahl** [cyberbotics/webots#1203]
**Describe the Bug**
Highway Driving example does not work properly due to sumo traffic simulation not starting

**Steps to Reproduce**
1.  Start Highway driving example

**Expected behavior**
sumo should start

**Console output**
Could not connect to TraCI server at localhost:1833 [Errno 61] Connection refused

./sumo --version
dyld: Library not loaded: /opt/X11/lib/libX11.6.dylib
  Referenced from: /Applications/Webots.app/projects/default/resources/sumo/bin/./sumo
  Reason: image not found
Abort trap: 6

**System**
 - Operating System: Mac OS 10.13.6 High Sierra
 - Graphics Card: Radeon RX 560 4096 MB

**Additional context**
XQuartz 1.18.4 (xorg-server 1.18.4)
MacPorts base version 2.6.0

**Comment by WolfgangFahl** [comment]
I am trying:

```
sudo port selfupdate
sudo port -v install xorg
```
according to https://www.xquartz.org/releases/

**Comment by fabienrohrer** [comment]
Could you uninstall xorg from port, and install it from the dmg? https://www.xquartz.org

This is the recommended way from our doc: https://cyberbotics.com/doc/automobile/sumo-interface

**Comment by WolfgangFahl** [comment]
I am rather happy with my macports xquartz which is now upgraded to 1.20.6 as part of macports 2.6.2.

The needed  dylib is at

```
find /opt/local/lib -name libX11.6.dylib 
/opt/local/lib/libX11.6.dylib
```
and somehow there seems to be hard coded library path /opt/X11/lib in the sumo version you are shipping.

So
```
sudo mkdir /opt/X11
sudo ln -s /opt/local/lib /opt/X11/lib
```
fakes the availability and:

./projects/default/resources/sumo/bin/sumo --version
SUMO Version 0.30.0

works.

**Comment by WolfgangFahl** [comment]
Looks good now ..

![grafik](https://user-images.githubusercontent.com/1336221/70919779-c3404480-2021-11ea-9ae2-85f7f2c81871.png)

**Comment by fabienrohrer** [comment]
Great! Well spotted.

**Comment by WolfgangFahl** [comment]
You might want to allow specifying the path to sumo as you do allow specifying the path to python. There is a newer sumo version available directly with macports ....

**Comment by fabienrohrer** [comment]
This is a good idea. We should upgrade SUMO on every OS too.

**Comment by DavidMansolino** [comment]
Updating at each release is costly for us (because we have to make a new package of SUMO to be  embed in Webots every 6 months).

However, what can easily be done which will make the interface way more scalable would be that if `SUMO_HOME` is defined (which is the case when you install SUMO manually) it will use the version of SUMO pointed by this environment variable.

**Comment by DavidMansolino** [comment]
The proposed solution has been implemented in #1233.
This will be included in the next version of Webots :smiley:
